development/docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
       - ./base_configuration.py:/opt/netbox/netbox/netbox/base_configuration.py
       - ./netbox_${NETBOX_VER}/configuration.py:/opt/netbox/netbox/netbox/configuration.py
       - ../netbox_onboarding:/source/netbox_onboarding
+      - ../:/source
```

Comment:
For Bandit to be able to get the config file. Any other file in the root file system will not get passed into the container except at build time without the volume.

Reply:
ok then please can you remove the line above that it not required anymore `- ../netbox_onboarding:/source/netbox_onboarding`

Reply:
If we remove the config file Bandit will error (at least previous experience) that it could not find the configuration file. So this should remain in place to complete the Bandit setup.